site/docs/components/react-resizable-panels-theme/usage.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Using the package
+
+## Import
+
+Make sure to install the React Resizable Panels package, `react-resizable-panels`.
+
+To install the `@salt-ds/react-resizable-panels-theme` package, run the command appropriate to your environment: `yarn add @salt-ds/react-resizable-panels-theme` or `npm install @salt-ds/react-resizable-panels-theme`
+
+### Import Salt React Resizable Panels theme
+
+To import the Salt React Resizable Panels theme, use:
+
+```js
+import "@salt-ds/react-resizable-panels-theme/index.css";
+```
+
+Then wrap your React Resizable Panel Group with the `react-resizable-panels-theme-salt` class name:
```

Comment:
Is there a naming conversion established with custom classes

- `react-resizable-panels-theme-salt`
- `resize-handle-border-<border>`
- `resize-handle-salt-variant-<variant>`
- `resizable-panel-salt-variant-<variant>`